app/services/authentication/providers/facebook.rb

```diff
@@ -47,6 +47,13 @@ def self.sign_in_path(**kwargs)
         )
       end
 
+      def image_url
+        token = auth_payload.credentials.token
+        uid = auth_payload.uid
+        request = HTTParty.get("https://graph.facebook.com/#{uid}?fields=picture.type(small)&access_token=#{token}")
+        request["picture"]["data"]["url"]
```

Comment:
is there any chance this could fail? What if for whatever reason the response is not `response.ok?` - should we have a fallback to a default image? 

Maybe we can do something like: 

```ruby
if response.ok?
  response.dig("picture", "data", "url")
else
  Users::ProfileImageGenerator.call
end
```

this way, worst case scenario, we use a fallback profile image